ui-admin/src/study/participants/participantList/facets/EnrolleeSearchFacets.tsx

```diff
@@ -1,110 +1,270 @@
-  }
+const KeywordFacet = ({ searchState, updateSearchState }: {
+  searchState: ParticipantSearchState,
+  updateSearchState: (field: keyof ParticipantSearchState, value: unknown) => void
+}) => {
+  return <div>
+    <div>
+      <input
+        className='form-control'
+        type="text"
+        value={searchState.basicSearch || ''}
+        placeholder='Search by name, email, or shortcode'
+        onChange={e => updateSearchState('basicSearch', e.target.value)}/>
+    </div>
+  </div>
+}
 
-  /** clear all resets everything to defaults */
-  const clearAll = () => {
-    updateFacetValues(facets.map(facet => newFacetValue(facet)))
-  }
+const EnrolleeFacet = ({ studyEnvContext, searchState, updateSearchState }: {
+  studyEnvContext: StudyEnvContextT,
+  searchState: ParticipantSearchState,
+  updateSearchState: (field: keyof ParticipantSearchState, value: unknown) => void
+}) => {
+  const subjectOptions = [
+    { label: 'Participant', value: true },
+    { label: 'Non-participant (e.g., proxy)', value: false }
+  ]
+  const consentedOptions = [
+    { label: 'Consented', value: true },
+    { label: 'Not consented', value: false }
+  ]
 
-  const defaultActiveFacets = facetValues.map(facetValue =>
-    facets.findIndex(facet => facetValue.facet.keyName === facet.keyName).toString())
 
+  const selectedSubjectOption = subjectOptions.find(o => o.value === searchState.subject)
+  const selectedConsentOption = consentedOptions.find(o => o.value === searchState.consented)
   return <div>
-    <button className="btn btn-secondary float-end" onClick={clearAll}>Clear all</button>
-    <Accordion defaultActiveKey={defaultActiveFacets} alwaysOpen flush>
-      {facets.map((facet, index) => {
-        const matchedValIndex = facetValues.findIndex(facetValue => facetValue.facet.keyName === facet.keyName &&
-          facetValue.facet.category === facet.category)
-        // matchedVal will be undefined if there is no user-specified non-default value for the facet.
-        // The FacetComponent will then render it as the default value
-        const matchedVal = facetValues[matchedValIndex]
-        return <Accordion.Item eventKey={index.toString()} key={index}>
-          <Accordion.Header>{facet.label}</Accordion.Header>
-          <Accordion.Body>
-            <FacetView facet={facet} facetValue={matchedVal}
-              updateValue={facetValue => updateFacetValue(facetValue, matchedValIndex)}/>
-          </Accordion.Body>
-        </Accordion.Item>
-      })}
-    </Accordion>
+    {studyEnvContext.currentEnv.studyEnvironmentConfig.acceptingProxyEnrollment
+      && <>
+        <label>User type</label>
+        <Select
+          options={subjectOptions}
+          isClearable={true}
+          value={selectedSubjectOption ? selectedSubjectOption : null}
+          onChange={selectedOption => {
+            updateSearchState('subject', selectedOption?.value)
+          }}
+        />
+      </>}
```

Comment:
I was not sure if this concept of `User Type` was the best way to introduce this concept to study staff. Open to wording changes here.

Reply:
works for me, we can see what demo says